xdsl/transforms/lower_mpi.py

```diff
@@ -742,7 +743,11 @@ def walker(op: Operation):
                                       len(module.body.blocks[0].ops))
 
 
-def lower_mpi(ctx: MLContext, module: builtin.ModuleOp):
+@dataclass
+class LowerMPIPass(OperationPass):
+
+    name = 'lower-mpi'
+
     # TODO: how to get the lib info in here?
     lib_info = MpiLibraryInfo()
```

Comment:
what info exactly do you need?

Reply:
I don't, this comment was there and is about the MPI lowering itself, not the refactoring of passes in general